src/earthengine/__tests__/ee_worker_utils.spec.js

```diff
@@ -1,4 +1,9 @@
-import { hasClasses, getHistogramStatistics } from '../ee_worker_utils.js'
+import {
+    hasClasses,
+    getHistogramStatistics,
+    getStartOfEpiYear,
+    getPeriodDates,
```

Comment:
Do you think it's worth adding tests for the other exported functions in this file?